geo/src/algorithm/area.rs

```diff
@@ -399,7 +399,8 @@ mod test {
             coord! { x: 0.0, y: 1.0 },
             coord! { x: 1.0, y: 0.0 },
         );
-        assert_relative_eq!(triangle.signed_area(), -0.5);
+        // triangles are always ccw, thus positive
+        assert_relative_eq!(triangle.signed_area(), 0.5);
```

Comment:
Test is redundant now since you can't check a triangle for negative area. Remove?

Reply:
Slight preference to keep it, if only that it documents that re-wind the input. But either is fine with me.